server/server-utils/emailProvider.js

```diff
@@ -0,0 +1,34 @@
+var nodemailer = require('nodemailer');
+
+function emailProvider() {
+
+  //define environment variables for e-mail
+  if (!process.env.APP_EMAIL || !process.env.APP_EMAILPW) {
```

Comment:
I like the idea of crashing if these aren't defined on startup for apps in `production` mode.